components/EpisodesList/JumpTopButton.js

```diff
@@ -14,7 +19,7 @@ export default function JumpTopButton({ onJumpTop }) {
 
 const JumpToTopButton = styled.button`
   position: fixed;
-  bottom: 8rem;
+  bottom: ${(props) => (props.showPlayer === true ? "8rem" : "5.5rem")};
```

Comment:
`bottom: ${(props) => (props.showPlayer ? "8rem" : "5.5rem")};` should also work 👀

Reply:
fixed